### PR TITLE
Kernel Events Proofreading after #4901

### PR DESCRIPTION
**Pull request by weaverryan**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3+ |
| Fixed tickets | n/a |

Hi guys!

This is a proofread of #4901. I removed the ticks in the titles - iirc, the ticks don't work in titles. I also fixed a few minor bugs I believe.

Thanks!

### CODE REVIEW

**Comment by WouterJ** [comment]
Literlas in the titles do work, but you can't reference them using `:ref:` or ``` ``_ ``` (but you can reference them in the url using `#kernel-request`)

**Comment by weaverryan** [comment]
Thanks @WouterJ - I re-added the literals. Look good otherwise?

File: reference/events.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Symfony Framework Events
 ========================
 
+When the Symfony Framework (or anything using the :class:`Symfony\\Component\\HttpKernel\\HttpKernel`)
+handles a request, a few core events are dispatched so that you can add listeners
+throughout the process. These are called the "kernel events". For a larger
+explanation, see :doc:`/reference/http_kernel/introduction`.
```
Comment (WouterJ) [inline reference/events.rst]: should be `/components/http_kernel/introduction` afaik

**Comment by WouterJ** [comment]
apart from the typo (causing the build failure), perfect!

**Comment by xabbuh** [comment]
:+1:

**Pull request merged**